### DISCUSSION TITLE
Varargs in C#

### DISCUSSION THREAD

**Discussion by jgh07** [dotnet/csharplang#8476] [Q&A]
I found the following in [this document](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/botr/clr-abi.md#varargs):
>Varargs refers to passing or receiving a variable number of arguments for a call.
>
>C# varargs, using the params keyword, are at the IL level just normal calls with a fixed number of parameters.
>
>Managed varargs (using C#'s pseudo-documented "...", __arglist, etc.) are implemented almost exactly like C++ varargs. [...]

What is meant by 'pseudo-documented "..."'? I know what ``__arglist`` is and I thought it was the only way to declare varargs methods in C#, but according to this document, it also supports the C syntax? I tried the following, but it does not compile.
````csharp
void Test(...) {}
````
Is this an error in the documentation or does this syntax actually exist?

**Comment by BlinD-HuNTeR** [comment]
No, this syntax doesn't exist. The paragraph you mentioned is speaking figuratively, you can read 'pseudo-documented "..."' as 'pseudo-documented "bananas"', followed by the statement that `__arglist` is C#'s "bananas".

**Reply by jgh07** [reply]
That's what I thought, but the comma and 'etc.' threw me off, suggesting there is more than just ``__arglist``.
